### DISCUSSION TITLE
GitHub360-Summer24 Discussions!

### DISCUSSION THREAD

**Discussion by planetoftheweb** [Pixel-Prowess/GitHub360-Summer24#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by DeiseFreire** [comment]
You are very intelligent, I hope someday I reach your level!

**Reply by planetoftheweb** [reply]
Very kind of you to say. A lot of my apparent skills is just trying and trying again without giving up. I know you already have that, so you're on your way!

**Reply by DeiseFreire** [reply]
I just subscribed to your YouTube channel. You are a highly successful man!

**Reply by DeiseFreire** [reply]
Could you share with me what your academic journey was like until you became a professor at LinkedIn Learnin? I would like to be inspired by your journey

**Reply by planetoftheweb** [reply]
I've been teaching as a lab aide way back from high school and while I was at Tribune Interactive, I started teaching at local community colleges, then found one of my friends from an earlier job who was working for a book publisher and got a deal for a book. I continued working in the industry and eventually Applied to teach at Lynda.com just through an online form (the company that LinkedIn bought) and I've been there for more than 12 years.

**Comment by anjaleeDS** [comment]
I would love to see some suggested codespace or project templates for data science work with jupyter notebooks.

**Comment by planetoftheweb** [comment]
Here's a template from my colleague Morten that has a Jupyter notebook setup:
https://github.com/LinkedInLearning/open-ai-api-multimodal-development-with-gpt-4o-5949017

Here's one that I use for python development when working with an AI like Llama
https://github.com/LinkedInLearning/developing-with-llama-3-meta-s-innovative-and-open-large-language-model-5950131

Next week, I'll make sure I cover how to set these up.

**Reply by rive-carl** [reply]
Best recommended practices to set these up?

**Reply by planetoftheweb** [reply]
You can hit the fork button to make a copy or download a zip file, which would remove the .git folder and let you start your own.

**Comment by anjaleeDS** [comment]
**_Ray_**, what are some of the best practices you would recommend for new class content creators?

**Reply by DeiseFreire** [reply]
Are you thinking about being a content creator?

**Reply by planetoftheweb** [reply]
That could be a whole class. It would depend a lot on what sort of content creation you'd be doing. YouTube? Classroom. I would say I started by recording my own videos for my classes and then making connections with others is what really allowed me to grow there.

Although I do create content on YouTube, it's not really something I'm heavily pursue. I'd ask what your goals are before I'd answer this further.

**Reply by DeiseFreire** [reply]
If you need my help creating subtitles, tool tips for presentations, I'll be happy to help.

**Reply by DeiseFreire** [reply]
You can look for me on LinkedIn so I can help you. I believe in virtual friendships

**Reply by DeiseFreire** [reply]
Some studies say that in video classes longer than 5 minutes, students begin to lose concentration. The longer the video class, the lower the participation, video classes longer than 20 minutes are considered long

**Reply by PhysicsPhun** [reply]
The typical university class used to be 52 minutes which if you work out the units comes out to a "microcentury" (which makes it sound really long).  Not sure it is a good commentary that students have very short attention spans.  It could be the impact of social media or tv because the commercials shorten attention.  And not sure we shouldn't as a society of learners go back to having longer attention spans.

**Reply by DeiseFreire** [reply]
Thank you for your interaction @PhysicsPhuna. Yes, I can agree with you , sometimes, depending on the complexity of the content, we need more time to develop a reasoning. I heard that the class structure in Chinese schools is 40 minutes of class and 10 minutes of break.

 > [!NOTE]
"typical learner’s attention span fades after 15 to 20 minutes for in-person teaching, and is as short as 6 to 8 minutes for online activities" <https://journals.sagepub.com/doi/full/10.1177/23821205211020760#bibr13-23821205211020760>

**Comment by planetoftheweb** [comment]
Interesting. Could be true. Do you have links to the studies, would be great to check that out. I definitely watch video content in pieces. I don't think it's the best method if all you're doing is watching videos. The interaction and doing is more important. That's why I try to always follow along or to give exercises or examples. You learn more from doing rather than watching.

**Reply by DeiseFreire** [reply]
Yes, the researchers say that it is better to alternate video classes, reading, exercises, writing activities, for example, 15 minutes of reading and interpreting a text, then 15 minutes of solving exercises, this way, the student's concentration can be stimulated and disturbances in the learning process are alleviated.

It is easier to maintain focus and concentration in short, frequent sessions than in one long session 
> [!NOTE]
"If you had an hour to learn something, you would retain more if you practiced it in four sessions of fifteen minutes each, spread over different days, than than if you spent an hour straight working without stopping." (Sprinthall & Sprinthall, 1993, p. 208).”

**Reply by DeiseFreire** [reply]
There is a scientific article that cites this channel here [The School of Life](https://www.youtube.com/@theschooloflifetv/videos) as a reference for a channel with short video classes.

**Reply by DeiseFreire** [reply]
I've already used this site to make some creations, it's very interesting. Maybe you know a friend of yours who is interested in this type of educational designer <https://www.animaker.co/video-maker>

Video lesson example: https://www.youtube.com/@aprendeespanolcontangerine

To create talking avatar for free <https://app.tokkingheads.com/avatar-select>

**Reply by anjaleeDS** [reply]
Thank you for the suggestions, everyone! I'm teaching a class, same format as Ray in the fall. It's 2 hours per lecture and I'm trying to change it up a bit so that students aren't dying by the end of the evening while also getting value and their money's worth. My course is Introduction to Data Visualization in Business with Python :)

Any suggestions would be appreciated.

Also, @planetoftheweb I actually just wrote a recommendation after watching one of your tutorials! Loved that it was self contained and 15 min long!!

**Reply by planetoftheweb** [reply]
Super. Good luck with your class and thanks for the recommendation.

**Reply by DeiseFreire** [reply]
I am taking a course at the Università di Napoli Federico II, in Italy. The course is remote and the teacher is a talking avatar created with Artificial Intelligence. I will leave some websites that work with talking avatars for you.
D-ID.com <https://www.d-id.com/>
Yepic.ai <https://www.yepic.ai/>
Vidnoz AI <https://pt.vidnoz.com/>
Hey Gen <https://www.heygen.com/>
VEED.IO <https://www.veed.io/>
Of these talking avatar creation sites, the only one I tested was Vidnoz AI, it offers free and paid features and I really liked it.

**Reply by DeiseFreire** [reply]
@anjaleeDS and @planetoftheweb suggestion: sign up as Udemy instructors

**Reply by planetoftheweb** [reply]
The AI generated teacher is a pretty hot topic for us right now. I'm curious what you thought of it. It seems a popular option for some content, but people are concerned about authenticity.

I personally think that those types of tools should be studied to see where they make sense, but that they're not necessarily the best.

**Reply by DeiseFreire** [reply]
Yes, I'm really enjoying it, thanks for asking. When I'm on the platform I'll make a print of the video class or film a little, I'll also look for how many students are taking the course with me, so I can publish it here so you can understand the graphic designer of the class

**Reply by DeiseFreire** [reply]
@planetoftheweb and @anjaleeDS I filmed a bit of the class and then turned it into a gif so I could show you the format of the class with a talking avatar.

<p>This is my avatar teacher from my English class at the Università di Napoli Federico II, in Italy.</p>
<img src="https://github.com/user-attachments/assets/265f1314-739f-4593-a58d-c72e8f177a87" alt="Gato animado" width="300" height="200">
